tests/inc/Config/HttpDataSourceTest.php

```diff
@@ -4,11 +4,11 @@
 
 use PHPUnit\Framework\TestCase;
 use RemoteDataBlocks\Tests\Mocks\MockDataSource;
-
+use WP_Error;
 class HttpDataSourceTest extends TestCase {
-	private MockDataSource $http_data_source;
+	private MockDataSource|WP_Error $http_data_source;
```

Comment:
This was necessary so that the error pathway for migrations could be tested.

@chriszarate  - could this become method scoped, or is there a plan down the line to add more tests that would benefit from this being class scoped?

Reply:
Not sure what you are referring to when you say method- or class-scoped.

Reply:
This `$http_data_source` property isn't very useful if each test is creating their own instance, it can just be removed and the tests updated to use a local variable.

Reply:
I hit that merge way too quickly. I'm going to make two follow up PRs in which I'll do that:

- Add some docs on how to do migration
- Add a migration for the `__version` property